### DISCUSSION TITLE
Dropout and MC Dropout for U-Net uncertainty quantification

### DISCUSSION THREAD

**Discussion by param1101** [Project-MONAI/MONAI#7295] [Q&A]
If I create a Unet model and use an inferer, does this automatically turn off dropout at inference?

If so, what can I do to turn on dropout at inference? I want to quantify the uncertainty in predictions using this but didn't really finding anything related to this

**Comment by KumoLiu** [comment]
You can set `dropout` by yourself.
https://github.com/Project-MONAI/MONAI/blob/d5585c34fc44f46af9ffb28522ce44c7ed869654/monai/networks/nets/unet.py#L69C8-L69C8
